### DISCUSSION TITLE
Airgap Install Issues

### DISCUSSION THREAD

**Discussion by Yeagernaut** [Security-Onion-Solutions/securityonion#12279] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

/

### RAM

/

### Storage for /

/

### Storage for /nsm

/

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Airgap suite: What is the proper way to mount the ISO in the Manager? This is an SNNV purchased from SOS that had 2.4.10 on it out of the box. After failing to install we were told that 2.4.10 had known installation issues with airgap networks. Per SO recommendation we have downloaded 2.4.40-20240116 from SOS onto a blu-ray disc, burned the file from disc onto a laptop, and moved the file to the SNNV Manager node using scp. What are the commands to mount and run the new ISO? Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TotieBash** [comment]
soup -f [ iso-location ]

**Comment by reyesj2** [comment]
You referenced having SO appliances, please reach out to us via Professional Services for us to assist you best. An appliance reimage is a bit different than a regular reimage.

**Reply by reyesj2** [reply, marked as answer]
This is being handled directly through professional services.
